reports/_setup.qmd

```diff
@@ -171,3 +173,64 @@ shorten_number <- function(x) {
   scales::dollar(x, accuracy = 1, scale = 1 / 1000, suffix = "K")
 }
 ```
+
+```{r}
```

Comment:
This entire section is just to populate some metadata variables that can be used in text, tooltips, etc throughout all the other documents.